CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Add go1.9.4 & 1.8.7 and default go1.9/go1.8 to them.
```

Comment:
Please drop the `*`. PS: I don't know why, but that was the decided languages team format.